as-ap-opticscorr/as_ap_opticscorr/chrom/main.py

```diff
@@ -427,78 +429,71 @@ def _callback_estimate_chrom(self, pvname, value, **kws):
-        fam_index = self._SFAMS.index(fam)
+        fam_index = self._SFAMS.index(_SiriusPVName(pvname).dev)
         self._sfam_check_opmode_sts[fam_index] = value
 
         # Change the third bit of correction status
-        opmode = self._sync_corr
-        val = (1 if any(s != opmode for s in self._sfam_check_opmode_sts)
-               else 0)
+        opmode = _PSConst.States.SlowRefSync if self._sync_corr \
+            else _PSConst.States.SlowRef
         self._status = _siriuspy.util.update_bit(
-            v=self._status, bit_pos=2, bit_val=val)
+            v=self._status, bit_pos=2,
+            bit_val=any(s != opmode for s in self._sfam_check_opmode_sts))
         self.driver.setParam('Status-Mon', self._status)
         self.driver.updatePVs()
 
     def _callback_sfam_ctrlmode_mon(self,  pvname, value, **kws):
-        ps = pvname.split(self._PREFIX_VACA)[1]
-        if value == 1:
-            self.driver.setParam('Log-Mon', 'WARN:'+ps+' is Local.')
+        if value != _PSConst.Interface.Remote:
+            self.driver.setParam('Log-Mon', 'WARN:'+pvname+' is not Remote.')
             self.driver.updatePVs()
 
-        fam = ps.split(':')[1].split('-')[1]
-        fam_index = self._SFAMS.index(fam)
+        fam_index = self._SFAMS.index(_SiriusPVName(pvname).dev)
         self._sfam_check_ctrlmode_mon[fam_index] = value
 
         # Change the fourth bit of correction status
-        val = (1 if any(s == 1 for s in self._sfam_check_ctrlmode_mon)
-               else 0)
         self._status = _siriuspy.util.update_bit(
-            v=self._status, bit_pos=3, bit_val=val)
+            v=self._status, bit_pos=3,
+            bit_val=any(s != _PSConst.Interface.Remote
+                        for s in self._sfam_check_ctrlmode_mon))
         self.driver.setParam('Status-Mon', self._status)
         self.driver.updatePVs()
 
     def _callback_timing_state(self, pvname, value, **kws):
         if 'Sexts:State' in pvname:
             self._timing_check_config[0] = value  # Enbl
         elif 'Sexts:EVGParam' in pvname:
-            self._timing_check_config[1] = (1 if value == 1 else 0)  # Chroms
+            self._timing_check_config[1] = (value == 1)  # Chroms
         elif 'Sexts:Pulses' in pvname:
-            self._timing_check_config[2] = (1 if value == 1 else 0)  # 1 pulse
+            self._timing_check_config[2] = (value == 1)  # 1 pulse
         elif 'Sexts:Duration' in pvname:
-            self._timing_check_config[3] = (1 if value == 0.15 else 0)  # 150us
+            self._timing_check_config[3] = (value == 0.15)  # 150us
         elif 'ChromsMode' in pvname:
-            self._timing_check_config[4] = (1 if value == 3 else 0)  # External
+            self._timing_check_config[4] = (value == 3)  # External
         elif 'ChromsDelay' in pvname:
-            self._timing_check_config[5] = (1 if value == 0 else 0)  # 0s
+            self._timing_check_config[5] = (value == 0)  # 0s
```

Comment:
The comment you have made in this line should be `# 0us`, to explicitly inform the PV unit.